docs/reference/announcements-release-notes/870/870-release-notes.md

```diff
@@ -132,16 +132,17 @@ Create and append tasks with resources available in the current project and proc
+| [File picker does not display the name of the uploaded file for completed tasks (#25443)](https://github.com/camunda/camunda/issues/25443)               | <p>File picker does not display the name of the uploaded file for completed tasks.</p><p><ul><li>**Affects:** Tasklist</li><li>**Workaround**: File name can be viewed in Operate.</li></ul></p>                                                                                                                                                                                                                                                                                                                                                                                |
+| [File upload fails to AWS storage due to non-standard space in filename (#28375)](https://github.com/camunda/camunda/issues/28375)                       | <p>File upload fails to AWS storage due to non UTF-8 whitespace character in filename.</p><p><ul><li>**Affects:** Tasklist</li><li>**Workaround**: When using AWS S3 storage, use UTF-8 compatible characters.</li></ul></p>                                                                                                                                                                                                                                                                                                                                                    |
+| [Failed to replay batch at 'LoggedEvent (#30810)'](https://github.com/camunda/camunda/issues/30810)                                                      | <p>When updating from `8.6.13` to `8.7.0`, Zeebe processing can stop after the update in some situations, where multi-instance elements are used.</p><p><ul><li>**Affects:** Zeebe</li><li>**Workaround**: This issue is fixed in `8.7.1`. When affected, going to `8.7.1` fully mitigates the issue. There is no risk of data loss.</li></ul></p>                                                                                                                                                                                                                              |
+| [Zeebe backups fail when using Self-Managed environment in Azure with Managed Identity](https://github.com/camunda/camunda/issues/30860)                 | <p>When performing Zeebe backups with Azure and Managed Identity in a self-managed environment, Zeebe fails to load the application default credentials that are provided at runtime, preventing backups from taking place.</p><p><ul><li>**Affects:** Zeebe</li><li>**Workaround**: When using Azure backups with in a SM environment, the configuration for the Azure store should be set as environment variables as detailed [here](/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore/#azure-backup-store) instead of using Managed Identity.</li></ul></p> |
```

Comment:
```suggestion
| [Zeebe backups fail when using Self-Managed environment in Azure with Managed Identity](https://github.com/camunda/camunda/issues/30860)                 | <p>When performing Zeebe backups with Azure and Managed Identity in a self-managed environment, Zeebe fails to load the application default credentials that are provided at runtime, preventing backups from taking place.</p><p><ul><li>**Affects:** Zeebe</li><li>**Workaround**: When using Azure backups with in a SM environment, the configuration for the Azure store should be set as environment variables as detailed [here](/self-managed/operational-guides/backup-restore/zeebe-backup-and-restore.md#azure-backup-store) instead of using Managed Identity.</li></ul></p> |
```